app.rb

```diff
@@ -4,17 +4,57 @@
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  It's categorized by apartments and tenants. It has string value sand numerical values.
   # What are the properties for each of the two types of hashes
 
+
 # Use enumerables to -
   # Print all the addresses for the apartments
+      appartments[:address]
```

Comment:
Make sure you're running each line of code or the whole file, and if you get any errors (like this line would trigger) to fix it before moving on!